operation/fhir-operation-bulkdata/src/test/java/com/ibm/fhir/operation/bulkdata/util/BulkDataExportUtilTest.java

```diff
@@ -429,7 +429,7 @@ public void testCheckAndValidateTypesWithExtraComma() throws FHIROperationExcept
         BulkDataExportUtil util = new BulkDataExportUtil();
         List<String> result = util.checkAndValidateTypes(ps, null);
         assertNotNull(result);
-        assertTrue(result.isEmpty());
+        assertFalse(result.isEmpty());
```

Comment:
I guess this is fine at this level (BulkDataExportUtil), but I hope an invalid parameter value like this is caught when we first accept the export request :-)

Reply:
It's for the case of absence of the parameter.  Now we return a subset instead of all of them.